### DISCUSSION TITLE
Control items visualization

### DISCUSSION THREAD

**Discussion by minaspapadim** [pyscada/PyScada#163] [Q&A] [answered]
I managed to enter a control item in a control panel and make it visible in my page as a widget. 
The problem I am facing is that the values of the selected variables are not being displayed.
When I add a chart the values of those variables are recorded properly.

There are 2 different devices. 1 with the onewire temperature sensor and the other one with GPIO17.

I also faced an issue on controling the same gpio as an output.

Am I doing something wrong here?

![image](https://github.com/user-attachments/assets/d5e2652c-6588-45b9-ad42-9099c7c32675)
![image](https://github.com/user-attachments/assets/32d280ea-0df8-455e-a9ee-2ee714f0cb36)

**Comment by clavay** [comment]
Can you run the `collectstatic` django command to be sure you have the last JS file and refresh your page without using the cache (`ctrl+F5`) ?

**Reply by minaspapadim** [reply]
The values are still not visible.

![image](https://github.com/user-attachments/assets/ad89e200-45e9-41a1-902e-a9a8cb13c42e)

![image](https://github.com/user-attachments/assets/fb58a292-43d2-4d03-8ac0-aaac05ad9a8e)

**Reply by clavay** [reply]
Do you have any message in the browser console ?

**Reply by minaspapadim** [reply]
The only strage thing is that the initiallization is still on going and it never stops.
![image](https://github.com/user-attachments/assets/38b941dd-4a43-46b9-8026-dc0cb960543c)

**Reply by clavay** [reply]
Yes I understand.
But what are the messages in the browser console ?

**Reply by minaspapadim** [reply]
![image](https://github.com/user-attachments/assets/4fbae741-19cc-4706-b3de-b47d61c8d0c3)

![image](https://github.com/user-attachments/assets/0e5bbb74-11f7-4fdf-ae0f-560cbe16adb1)

**Reply by clavay** [reply, marked as answer]
Can you try this [fix](https://github.com/clavay/PyScada/tree/momentCheck) ? (install it, copy the static files and `ctrl+F5` to refresh the page)

**Reply by minaspapadim** [reply]
It fixed the issue!
![image](https://github.com/user-attachments/assets/468ac8fc-aa20-47fb-b1fe-f4ba767a478a)
